linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_tally2/oval/shared.xml

```diff
@@ -12,13 +12,28 @@
   check="all" check_existence="all_exist"
   comment="Verify deny configuation of pam_tally2" version="1">
     <ind:object object_ref="object_accounts_passwords_pam_tally2_deny_auth" />
+    <ind:state state_ref="state_var_accounts_passwords_pam_tally2_deny_value_upper_bound" />
+    <ind:state state_ref="state_var_accounts_passwords_pam_tally2_deny_value_lower_bound" />
   </ind:textfilecontent54_test>
 
   <ind:textfilecontent54_object id="object_accounts_passwords_pam_tally2_deny_auth" comment="Check deny configuration of pam_tally2" version="1">
     <ind:filepath>/etc/pam.d/common-auth</ind:filepath>
-    <ind:pattern operation="pattern match">^\s*auth(?:(?!\n)\s)+required(?:(?!\n)\s)+pam_tally2.so(?:(?!\n)\s)+(((?:(?!\n)\s)?[^\n]+)?onerr=fail((?:(?!\n)\s)+[^\n]+)?(?:(?!\n)\s)+deny=[123]((\s+\S+)*\s*$))|(((?:(?!\n)\s)?[^\n]+)?deny=[123]((?:(?!\n)\s)+[^\n]+)?(?:(?!\n)\s)+onerr=fail((\s+\S+)*\s*$))</ind:pattern>
+    <ind:pattern operation="pattern match">^\s*auth(?:(?!\n)\s)+required(?:(?!\n)\s)+pam_tally2.so(?:(?!\n)\s)+(?:(?:(?:(?!\n)\s)?[^\n]+)?onerr=fail(?:(?:(?!\n)\s)+[^\n]+)?(?:(?!\n)\s)+deny=([\d]+)(?:(?:\s+\S+)*\s*$))|(?:(?:(?:(?!\n)\s)?[^\n]+)?deny=([\d]+)(?:(?:(?!\n)\s)+[^\n]+)?(?:(?!\n)\s)+onerr=fail(?:(?:\s+\S+)*\s*$))</ind:pattern>
```

Comment:
The brackets in `[\d]+` are superfluous, you can use just `\d+`.

Reply:
done, thanks!